spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/plugin/GravitinoDriverPlugin.java

```diff
@@ -69,6 +75,15 @@ private void registerGravitinoCatalogs(SparkConf sparkConf, Set<String> catalogN
         });
   }
 
-  // Todo inject Iceberg extensions
-  private void registerSqlExtensions() {}
+  private void registerSqlExtensions(SparkConf conf) {
```

Comment:
how are you verifying that `GRAVITINO_DRIVER_EXTENSIONS` is injected to spark really?

Reply:
Maybe we can add a check after `registerSqlExtensions(conf);`. WDYT? @FANNG1

Reply:
Or add a test case to `SparkIcebergCatalogIT`. Get the sparkSession instance, and check whether `spark.sql.extensions` is included in sparkConf and whether `GRAVITINO_DRIVER_EXTENSIONS` is included in the parameter value

Reply:
i prefer the second one.

Reply:
> i prefer the second one.

+1

Reply:
Fixed